runtime/src/stakes.rs

```diff
@@ -106,15 +112,6 @@ impl Stakes {
                     |v| v.0,
                 );
 
-                // count any increase in points, can only go forward
-                let old_credits = old
-                    .and_then(|(_stake, old_account)| VoteState::credits_from(old_account))
-                    .unwrap_or(0);
-
-                let credits = VoteState::credits_from(account).unwrap_or(old_credits);
-
-                self.points += credits.saturating_sub(old_credits) * stake;
```

Comment:
nits: same with [this](https://github.com/solana-labs/solana/pull/10914/files#r451941881)